src/workflow/ArcanistLandWorkflow.php

```diff
@@ -942,6 +942,82 @@ private function findRevision() {
         pht('Accepted'));
     }
 
+    // Check if all paths were reviewed by reviewers listed on METADATA files.
+    // If this check throws an exception - silently pass.
+    try {
+      $uber_metadata_unreviewed_paths = $this->getConduit()->callMethodSynchronous(
+        'uber_metadata.unreviewed_paths',
+        array(
+          'revisionid' => $rev_id,
+        ));
+      if (array_key_exists('pass', $uber_metadata_unreviewed_paths)
+        && false === $uber_metadata_unreviewed_paths['pass']
+      ) {
+
+        id(new PhutilConsoleWarning('WARNING', pht(
+          'Revision contains paths which were not reviewed by METADATA '.
+          'reviewers. Likely land operation will be blocked.'
+        )))->draw();
+
+        foreach ($uber_metadata_unreviewed_paths['groups'] as $group) {
```

Comment:
What about `users`?

Reply:
This is "reviewers group" array which contains paths and reviewers. Meanwhile reviewers contains ldap groups and users. I've added JSON example to the comment to make this flow more visible.